lib/legato/management/finder.rb

```diff
@@ -14,10 +15,31 @@ def all(user, path=default_path)
           base_uri + path
         end
 
-        json = user.access_token.get(base_uri + path).body
-        items = MultiJson.decode(json).fetch('items', [])
-        items.map {|item| new(item, user)}
+        collect_items(user, uri).map do |item|
+          new(item, user)
+        end
       end
+
+      private
+        def items_key; "items"; end
+        def next_link_key; "nextLink"; end
+
+        def collect_items(user, base_uri)
+          next_uri = base_uri
+          item_collection = []
+
+          while next_uri
```

Comment:
I wonder if we shouldn't make this an option, since it technically changes the output of the method